tests/fixtures/buggy-app/views/checkout.html

```
@@ -0,0 +1,27 @@
+<!doctype html>
+<html lang="en">
+<head><meta charset="utf-8"><title>Checkout</title></head>
+<body>
+  <h1>Checkout</h1>
+  <form id="f">
+    <label>Email <input name="email" tabindex="1"/></label>
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)